doc/reference/storage_dir.md

```diff
@@ -1,25 +1,38 @@
 (storage-dir)=
 # Directory - `dir`
 
- - While this backend is fully functional, it's also much slower than
-   all the others due to it having to unpack images or do instant copies of
-   instances, snapshots and images.
- - Quotas are supported with the directory backend when running on
-   either ext4 or XFS with project quotas enabled at the filesystem level.
+The directory storage driver is a basic backend that stores its data in a standard file and directory structure.
+This driver is quick to set up and allows inspecting the files directly on the disk, which can be convenient for testing.
+However, LXD operations are not optimized for this driver.
 
-## Storage pool configuration
+## `dir` driver in LXD
+
+The `dir` driver in LXD is fully functional and provides the same set of features as other drivers.
+However, it is much slower than all the other drivers because it must unpack images and do instant copies of instances, snapshots and images.
+
+Unless specified differently during creation (with the `source` configuration option), the data is stored in the `/var/snap/lxd/common/lxd/storage-pools/` directory.
```

Comment:
The location is probably different for non-snap installations?

Reply:
Yes it will be /var/lib/lxd/storage-pools/....